third-culture-software/bhima
# Fixed Excel export issues for the Lots registry

Fixed a few issues in exporting the Lots registry as an Excel file.  Added more (initially hidden) fields and purged other (unneded) fields from the exported Excel file.  This also fixes the errors when some of the fields were converted to Excel output (some date values were invalid; converted the invalid date to an empty string).

Closes https://github.com/IMA-WorldHealth/bhima/issues/6295
Closes https://github.com/IMA-WorldHealth/bhima/issues/6296

**TESTING**
- Start with bhima_test
- Try Stock > Stock Lots (for the lot registry)
   - Use Menu > Columns and enable some of the new fields including 'Status'.  Check that the data in those new columns in the grid are okay.
   - Use Menu > Export to Excel and examine the output file:  Are the column labels good? Is the content of the columns okay?
- Try doing the same with a production data set (in particular check 'status' for correct handling of different status values).
